### ISSUE TITLE
LIGO module?

### ISSUE THREAD

**Issue by bsipocz** [astropy/astroquery#1260]
I'm not sure whether it would make any sense to provide access to it here, but it seems that they are making the data public via https://www.gw-openscience.org/ 

@duncanmmacleod what do you think?

**Comment by duncanmmacleod** [comment]
@bsipocz, yes, there is an increasing quantity of data being made available on gw-openscience.org, but mainly in the format of 'raw' data files. I don't have much experience with astroquery, but what could be made available fairly easily is a query interface to the 'event database'.

There is a JSON API to gw-openscience.org that allow for querying simple information about the published events, so I could throw something together if you can point me at the docs for what the output of an astroquery query should look like.

(apologies for the delay, travelling.)

**Comment by duncanmmacleod** [comment]
Ok, after a long delay, I'm ready to think about this properly. There exists a python API to the Gravitational-wave Open Science Center (GWOSC) data releases, [`gwosc`](https://gwosc.readthedocs.io), so I would wrap that into an astroquery module. Is it kosher to add (optional) dependencies to astroquery? (that package is itself pure python and only depends on `six`).

I will have a go at following the [template](https://astroquery.readthedocs.io/en/latest/template.html) and see what I can come up with.

**Comment by keflavich** [comment]
Generally, we like to avoid additional dependencies, but we can allow some module-specific runtime dependencies.  They should be imported at the class or method level, though, not at the module level, so they don't become global dependencies of astroquery.

**Comment by bsipocz** [comment]
Thank you @duncanmmacleod, this sounds great. As @keflavich said we prefer the dependencies the other way around (that packages depend on astroquery for their data access), but starting off the way you've suggested is also good for now.
